UDV-Core/src/Extensions/Contribute/src/CreateDocument.js

```diff
@@ -0,0 +1,243 @@
+export function CreateDocument(creationContainer, contributeController){
+
+  this.contributeController = contributeController;
+  this.creationContainer = creationContainer;
+
+this.myUploadedImage;
+  var docCreateButton = document.createElement('button');
+  docCreateButton.id = "docCreateButton";
+  var text = document.createTextNode("Create");
+  docCreateButton.appendChild(text);
+  document.getElementById("researchWindowTabs").appendChild(docCreateButton);
```

Comment:
I don't think we need to add this button to the Document research window since it's already in the document navigator.

Reply:
I think it is good to give the possibility to 'create' also from the browser, because the research window can be closed.

Reply:
Button has been removed